src/fabric/CircleNativeComponent.ts

```diff
@@ -0,0 +1,56 @@
+import codegenNativeComponent from 'react-native/Libraries/Utilities/codegenNativeComponent';
+import type { ViewProps, ColorValue } from 'react-native';
+import type {
+  Float,
+  Int32,
+  WithDefault,
+  // @ts-ignore TODO: remove once there is a .d.ts file with definitions
```

Comment:
DefinitelyTyped has annotations for react-native/Libraries/Types/CodegenTypes, here:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ffc894660966251e61928e62fffddb4705059ae4/types/react-native/Codegen.d.ts#L31-L61

Do we need this ts-ignore?